### PR TITLE
ELMO prod deployment PR

### PR DESCRIPTION
**Pull request by vthripur**
Here is pull request to move ELMO to prod
Jira Ticket: https://avantinc.atlassian.net/browse/DATA-10045

### CODE REVIEW

File: sql/dml_statements/dag_object_map.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO campaign.dag_object_map
+(campaign_type_config_id, trigger_dag, task_name, check_dependency, is_active, start_date, end_date)
+VALUES
+(2, 'Direct_Mail_2_3_selection', 'scoring', false, true, '2020-09-13 11:20:32.000', '9999-01-01 00:00:00.000'),
+(2, 'Direct_Mail_2_4_first_mail_publish', 'approval', false, true, '2020-09-13 11:20:32.000', '9999-01-01 00:00:00.000'),
+(3, 'Direct_Mail_Remail_3_3_acxiom_publish', 'approval', false, true, '2020-09-13 11:20:32.000', '9999-01-01 00:00:00.000');
```
Comment (scottshepard) [inline sql/dml_statements/dag_object_map.sql:6]: Is this missing an ELMO dag object map?

**Review by scottshepard — APPROVED** [review]
Approved conditional on two comments being resovled.

File: sql/dml_statements/dag_object_map.sql (hunk 1 above)
Comment (vthripur) [reply sql/dml_statements/dag_object_map.sql:6]: This is from lambda to trigger automatically when the file drops in respective buckets. This is not required for ELMO

File: dag_template/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from .dag_template import *
+from .acxiom_dag_template import *
```
[the remark this answers is not in the document]
Comment (vthripur) [reply dag_template/__init__.py:2]: Sorry forgot let me remove it

**Pull request merged**